app/src/main/java/com/example/weatherapp/models/Location.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.weatherapp.models
+
+class Location(
+    val name: LocationData
```

Comment:
I think a better approach here is mapping the `LocationData` properties to a `Location` class with same properties.

Here we are just wrapping the `LocationData` inside `Location`. It seems there is no benefit from layer separation point of view

Reply:
Changed the LocationData object to be called "LocationName"